### PR TITLE
Newbranch

### PR DESCRIPTION
**Pull request by bouteina-kadiri**

### CODE REVIEW

File: angular-techDraft/src/app/about/about.component.css [hunk 1]
```diff
@@ -0,0 +1,50 @@
+body {font-family: Arial, Helvetica, sans-serif;
+margin: 0;
+padding: 0;}
+
```
Comment (jp-2019) [inline angular-techDraft/src/app/about/about.component.css]: curly bracket should be on new line

File: angular-techDraft/src/app/about/about.component.css [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+section{
+
```
Comment (jp-2019) [inline angular-techDraft/src/app/about/about.component.css:9]: remove extra spaces

File: angular-techDraft/src/app/about/about.component.css [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+div{
+  background:pink;
+  margin: auto;
+  width:100px;
+}
+.header {
+  width: 70%;
+  background-color:lightgray;
+  padding : 20px 40px 10px 50px;
+
```
Comment (jp-2019) [inline angular-techDraft/src/app/about/about.component.css]: remove extra spacing

File: angular-techDraft/src/app/about/about.component.css [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  color: white;
+  text-decoration: none;
+  font-size: 24px;
+}
+
+.header a:hover {
+  background-color:plum;
+  
+}
+
```
Comment (jp-2019) [inline angular-techDraft/src/app/about/about.component.css:42]: remove extra spacing

**Review by jp-2019 — CHANGES_REQUESTED** [review]
tab component files should not be deleted.

File: angular-techDraft/src/app/about/about.component.css [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+section{
+
+  background: white;
+  color: gray;
+  padding:0px;
+  display:flex;
+  flex-direction:row;
+}
+div{
```
Comment (shakirullah7892) [inline angular-techDraft/src/app/about/about.component.css]: Spacing needed between classes and div. Put spacing before curly brackets.

File: angular-techDraft/src/app/about/about.component.css (hunk 3 above)
Comment (shakirullah7892) [inline angular-techDraft/src/app/about/about.component.css]: Header should be closed with the curly bracket.

File: angular-techDraft/src/app/about/about.component.spec.ts [hunk 6]
```diff
@@ -1,19 +1,20 @@
 import { async, ComponentFixture, TestBed } from '@angular/core/testing';
-import { TabComponent } from './tab.component';
```
Comment (shakirullah7892) [inline angular-techDraft/src/app/about/about.component.spec.ts]: Tab component was deleted. You are removing Tab Component.

File: angular-techDraft/src/app/about/about.component.spec.ts [hunk 7]
```diff
@@ -1,19 +1,20 @@
...
-  let fixture: ComponentFixture<TabComponent>;
+import { AboutComponent } from './about.component';
+
+describe('AboutComponent', () => {
+  let component: AboutComponent;
+  let fixture: ComponentFixture<AboutComponent>;
 
   beforeEach(async(() => {
     TestBed.configureTestingModule({
-      declarations: [ TabComponent ]
```
Comment (shakirullah7892) [inline angular-techDraft/src/app/about/about.component.spec.ts]: You are deleting the tab component from the declarations too.

File: angular-techDraft/src/app/about/about.component.css (hunk 4 above)
Comment (bouteina-kadiri) [reply angular-techDraft/src/app/about/about.component.css:42]: done

File: angular-techDraft/src/app/about/about.component.spec.ts (hunk 6 above)
Comment (bouteina-kadiri) [reply angular-techDraft/src/app/about/about.component.spec.ts]: re added tab component, this is a change in About component

**Review by jp-2019 — APPROVED** [review]

**Review by shakirullah7892 — APPROVED** [review]
All good.

**Pull request merged**